experimental/e2e/storage/types.go

```diff
@@ -52,7 +52,7 @@ func (e *Entry) Match(incoming *http.Request) *http.Response {
 
 // Storage is an interface for storing Entry objects.
 type Storage interface {
-	Add(*http.Request, *http.Response)
+	Add(*http.Request, *http.Response) error
 	Delete(*http.Request) bool
 	Load() error
 	Save() error
```

Comment:
Do you think it's still worth keeping `Save() error` as part of the API?

Reply:
good question. i was debating removing it as part of this change, and i think it can be safely removed at this point

Reply:
removed in [a04c90c](https://github.com/grafana/grafana-plugin-sdk-go/pull/491/commits/a04c90c45484a0c7046d177b47ad6f431045e072)